apps/example-tailwind/src/app/app.routes.ts

```diff
@@ -33,15 +33,15 @@ const authRoutes: Route[] = [
             import('@myorg/app-example-auth').then(
                 (m) => m.PageAccountCreatedComponent,
             ),
-        resolve: { layout: setLayout(PageLayout.Center) },
+        resolve: { layout: setLayout(PageLayout.Cta) },
```

Comment:
why use cta? remove it please use the center
```suggestion
        resolve: { layout: setLayout(PageLayout.center) },
```

Reply:
I think Cta & center has two different layouts. CTA should have a header navbar, but center should not.